src/test/java/uk/gov/hmcts/reform/roleassignment/helper/TestDataBuilder.java

```diff
@@ -533,7 +533,7 @@ public static ExistingRoleAssignment buildExistingRoleForIAC(String actorId, Str
                                  .actorIdType(ActorIdType.IDAM)
                                  .classification(Classification.PUBLIC)
                                  .readOnly(true)
-                                 .status(CREATE_REQUESTED)
+                                 .status(CREATED)
```

Comment:
why you change it? we are checking the status CREATED_REQUESTED in drool rule.

Reply:
Done